test/index.js

```diff
@@ -39,7 +30,7 @@ serial('Measure field perf for site in CruX', async t => {
 
 serial('Measure field perf for site site not in CruX', async t => {
   const resName = 'not-in-field.json'
-  await runLighthouse('https://example.com/', { ...lhOptions, outputPath: `./results/${resName}` })
+  await runLighthouse('https://alekseykulikov.com/', { ...lhOptions, outputPath: `./results/${resName}` })
```

Comment:
Do you think results always will be stable on Field and match snapshot? Our intention to stub PSI response to not update snapshot for each new CI run or PSI cache invalidation hence new perf measurements of the site.